librz/include/rz_core.h

```diff
@@ -892,12 +892,14 @@ RZ_API void rz_core_bin_print_source_line_info(RzCore *core, const RzBinSourceLi
 RZ_API bool rz_core_sym_is_export(RZ_NONNULL RzBinSymbol *s);
 
 // bin_dwarf
-RZ_API void rz_core_bin_dwarf_print_abbrev_section(const RzBinDwarfDebugAbbrev *da);
-RZ_API void rz_core_bin_dwarf_print_attr_value(const RzBinDwarfAttrValue *val);
-RZ_API void rz_core_bin_dwarf_print_debug_info(const RzBinDwarfDebugInfo *inf);
-RZ_API void rz_core_bin_dwarf_print_loc(HtUP /*<offset, RzBinDwarfLocList *>*/ *loc_table, int addr_size);
-RZ_API void rz_core_bin_dwarf_print_aranges(RzList /*<RzBinDwarfARangeSet *>*/ *aranges);
-RZ_API void rz_core_bin_dwarf_print_line_units(RzList /*<RzBinDwarfLineUnit *>*/ *lines);
+RZ_API char *rz_core_bin_dwarf_abbrev_decl_dump(RzBinDwarfAbbrevDecl *decl);
```

Comment:
I wonder if it's better to do `_as_str()` or `_to_str()` instead of `_dump()`, like we did in some other places. @ret2libc @thestr4ng3r @wargio opinions?